blocksuite/tests-legacy/e2e/selection/block.spec.ts

```diff
@@ -696,7 +700,8 @@ test('should keep selection state when scrolling forward with the scroll wheel',
   expect(scrollTop1).toBe(0);
 });
 
-test('should not clear selected rects when clicking on scrollbar', async ({
+// TODO(@fundon): should remove it
+test.skip('should not clear selected rects when clicking on scrollbar', async ({
```

Comment:
Why do you skip these two test cases?

Reply:
Clicking on the scrollbar is actually likely to be outside the editor.
https://github.com/toeverything/AFFiNE/pull/10507/files#diff-db0a5ecfe107fb09adfc7df9674ae7246d3a4c8b15b6740725cf0bad4e07e801R263